primitives/block/src/equivocation_proof.rs

```diff
@@ -360,36 +360,36 @@ impl DoubleProposalProof {
 
     /// Network ID this equivocation happened on.
     pub fn network(&self) -> NetworkId {
-        self.header1.network
+        self.proposal1.proposal.network
     }
     /// Address of the offending validator.
     pub fn validator_address(&self) -> &Address {
         &self.validator_address
     }
     /// Block number at which the offense occurred.
     pub fn block_number(&self) -> u32 {
-        self.header1.block_number
+        self.proposal1.proposal.block_number
     }
     /// Round of the proposals.
     pub fn round(&self) -> u32 {
-        self.header1.round
+        self.proposal1.round
     }
     /// Hash of header number 1.
     pub fn header1_hash(&self) -> Blake2bHash {
-        self.header1.hash()
+        self.proposal1.proposal.hash()
     }
     /// Hash of header number 2.
     pub fn header2_hash(&self) -> Blake2bHash {
-        self.header2.hash()
+        self.proposal2.proposal.hash()
     }
 
     /// Verify the validity of a double proposal proof.
     pub fn verify_excluding_address_and_network(
         &self,
         signing_key: &SchnorrPublicKey,
     ) -> Result<(), EquivocationProofError> {
-        let hash1: Blake2bHash = self.header1.hash();
-        let hash2: Blake2bHash = self.header2.hash();
+        let hash1: Blake2bHash = self.proposal1.proposal.hash();
+        let hash2: Blake2bHash = self.proposal2.proposal.hash();
```

Comment:
As far as I can see these are now only used for sorting. Sorting could also just use the Blake2sHash instead saving one hashing.